lib/parser.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+const path = require('path')
+
+module.exports = class Parser {
+
+  static commandParser (opts) {
+    const command = opts.command
+    if (!command) return null
+    if (Array.isArray(command)) {
+      return command.join(' ')
+    }
+    return command
+  }
+
+  static buildDockerfilePathParser (opts) {
+    const build = opts.build
+    const warnings = opts.warnings
+    let dockerFileName = 'Dockerfile'
+    if (!build) return null
+
+    if (build.args) {
+      // NOTE: Maybe pass these to the ENVs?
```

Comment:
Just pointing out that we could potentially support this if we passed the `build.args` to the ENVs, which we currently do.